torch/csrc/jit/codegen/cuda/compute_at.cpp

```diff
@@ -400,13 +501,6 @@ unsigned int ComputeAt::backwardComputeAt_impl(
     }
 
     consumer->setMaxProducer(consumer_compute_at_pos);
-    for (auto other_consumer : ir_utils::consumerTvsOf(producer)) {
-      if (other_consumer != consumer) {
-        auto max_consumer_pos =
-            getConsumerPosAlignedToProducerCA(other_consumer, producer);
-        other_consumer->setMaxProducer(max_consumer_pos);
-      }
-    }
```

Comment:
Why is this deleted?

Reply:
Hmm, it seems unnecessary, but wasn't there some cases it was actually necessary? Here's the PR that introduced this logic. #1035